object-cache.php

```diff
@@ -511,14 +511,23 @@ function switch_to_blog( $blog_id ) {
 	function colorize_debug_line( $line ) {
 		$colors = array(
 			'get' => 'green',
+			'get_multi' => 'green',
 			'set' => 'purple',
 			'add' => 'blue',
 			'delete' => 'red',
 		);
 
 		$cmd = substr( $line, 0, strpos( $line, ' ' ) );
 
-		$cmd2 = "<span style='color:{$colors[$cmd]}'>$cmd</span>";
+		// Start off with a neutral default color...
+		$color_for_cmd = 'brown';
+
+		// And if the cmd has a specific color, use that instead
+		if ( isset( $colors[ $cmd ] ) ) {
+			$color_for_cmd = $colors[ $cmd ];
+		}
+
+		$cmd2 = "<span style='color:{$color_for_cmd}'>$cmd</span>";
```

Comment:
Should we escape this just to be safe?

Reply:
Good call - The WP functions should be safe to use here b/c this only runs in the `stats()` function, which presumably is only ever called when displaying stats to a user (so WP will have loaded by then). 359daa6